docs/sources/setup-grafana/configure-docker.md

```diff
@@ -8,45 +8,70 @@ keywords:
+
+## Ubuntu image
+
+**Grafana Enterprise**: `grafana/grafana-enterprise:<version>-ubuntu`
 
-### Run Grafana container using bind mounts
+**Grafana Open Source**: `grafana/grafana-oss:<version>-ubuntu`
 
-You may want to run Grafana in Docker but use folders on your host for the database or configuration. When doing so, it becomes important to start the container with a user that is able to access and write to the folder you map into the container.
+These images are based on [Ubuntu](https://ubuntu.com/), available in the [Ubuntu official image](https://hub.docker.com/_/ubuntu). It is an alternative image for those who prefer an Ubuntu-based image and/or are dependent on certain tooling not available for Alpine.
+
+## Run a specific version of Grafana
+
+You can also run a specific version, or run a beta version based on the main branch of the [grafana/grafana GitHub repository](https://github.com/grafana/grafana).
+
+> **Note:** If you are on a Linux system (e.g. Debian, Ubuntu), you might need to add `sudo` before the command or add your user to the `docker` group by following the official documentation to [run Docker without a non-root user](https://docs.docker.com/engine/install/linux-postinstall/).
+
+
+To run a specific version of Grafana mention it in the command <version number> section:
 
 ```bash
-mkdir data # creates a folder for your data
-ID=$(id -u) # saves your user id in the ID variable
+docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:<version number>
+```
 
-# starts grafana with your user id and using the data folder
-docker run -d --user $ID --volume "$PWD/data:/var/lib/grafana" -p 3000:3000 grafana/grafana-enterprise:8.2.1
+Example:
+
+This command will run Grafana Enterprise container which has version 9.4.7. If you want to run a different version e.g. 9.4.3 then just put it in the version-number section
+
+```bash
+docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:9.4.7
 ```
 
-## Default paths
+## Run the Grafana main branch
+
+For every successful build of the main branch, we update the `grafana/grafana-oss:main` and `grafana/grafana-oss:main-ubuntu` tags. Additionally, two new tags are created, `grafana/grafana-oss-dev:<version>-<build ID>pre` and `grafana/grafana-oss-dev:<version>-<build ID>pre-ubuntu`, where version is the next version of Grafana and build ID is the ID of the corresponding CI build. You can use these Tags to get access to the latest main builds of Grafana.
+When running Grafana main in production, we strongly recommend that you use the `grafana/grafana-oss-dev:<version>-<build ID>pre` tag. This tag guarantees that you use a specific version of Grafana instead of the most recent commit.
```

Comment:
```suggestion
To ensure stability and consistency, we strongly recommend using the `grafana/grafana-oss-dev:-pre` tag when running the Grafana main branch in a production environment. This tag ensures that you are using a specific version of Grafana instead of the most recent commit, which could potentially introduce new bugs or issues.
```

Reply:
This section also needs to be rechecked by a developer as I cannot find "dev" tag on the docker hub. Only "develop" i.e.
https://hub.docker.com/layers/grafana/grafana-oss/develop/images/sha256-9fb294499601d06fe643f1a8203cf46fd17c4e76e28869a755b6119231a314e4?context=explore

Reply:
dev isn't a tag, there is the grafana-oss image and a separate grafana-oss-dev image (and the same arrangement for enterprise). This avoids polluting the tag namespace for the main grafana images with thousands of pre-release tags